### ISSUE TITLE
Is there a reason JIRA plugin must assume Basic Auth for activities? 

### ISSUE THREAD

**Issue by madorb** [RoadieHQ/roadie-backstage-plugins#277]
I'm able to successfully pull back issues with:

```
  '/jira/api':
    target: https://my-jira-instance.com
    headers:
      Authorization: 'Bearer <my-token>'
```

but then when it attempts to get activities: it appears to force the use of basic auth, which, of course, doesn't work as we don't have a basic auth value to send. I tried with basic auth, but could not get it to work at all, just got 401s.

https://github.com/RoadieHQ/roadie-backstage-plugins/blob/6d7c927791118d0bb980207f3dfa4188de9ddfef/plugins/frontend/backstage-plugin-jira/src/api/index.ts#L179

If not, it would be better to clarify in the documentation that basic auth is a hard requirement

**Comment by Irma12** [comment]
Hey @madorb thank you for bringing this up. Does it work when you omit '&os_authType=basic` in the request? If so, maybe you could set up a PR so we can take a look? :)

**Comment by madorb** [comment]
it does seem to work when i access it via:
`http://localhost:7000/api/proxy/jira/api/activity?maxResults=25&streams=key+IS+MYPROJECT`
and gets a 401 with:
`http://localhost:7000/api/proxy/jira/api/activity?maxResults=25&streams=key+IS+BP&os_authType=basic`

I suppose you'd need to look at the `Authorization` type in the configuration and if it includes `Bearer` then exclude the attribute, else include it.

**Comment by Irma12** [comment]
Hey @madorb sorry for silence from our side regarding this. 

Using Basic auth is required to retrieve activity in case you want to view restricted content (https://developer.atlassian.com/server/framework/atlassian-sdk/consuming-an-activity-streams-feed/#authentication) but we added the ability to go with Bearer token. This is included in new release (🎉 1.3.0). 

Please, keep in mind you need to add 'jira/token-type: Bearer' annotation to specify if you are using Bearer token, as included in Readme files for the plugin :)
